support/windows-server/networking/slow-smb-file-transfer.md

```diff
@@ -15,11 +15,17 @@ Server Message Block (SMB) file transfer speeds can slow down depending on the s
 
 ## Slow transfer
 
+> [!NOTE]
+> SMB Signing and SMB Encryption is known to reduce SMB transfer speeds. The amount of performance loss depends greatly on the capabilities of the hardware involved.
```

Comment:
signing and encryption are lower case

Reply:
If you want to give them a hint, it's mainly CPU core count, speed, and utilization from other workloads